components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/HTTPGetFunctionImpl.java

```diff
@@ -64,6 +63,17 @@ public void httpGet(String endpointURL, Object... params) {
                             "and eventHandlers (Map<String, Object>) respectively.");
                 }
                 break;
+            case 3:
+                if (params[0] instanceof Map && params[1] instanceof Map && params[2] instanceof Map) {
+                    headers = validateHeaders((Map<String, ?>) params[0]);
+                    authConfig = getAuthConfigModel((Map<String, Object>) params[1]);
+                    eventHandlers = (Map<String, Object>) params[2];
+                }  else {
+                    throw new IllegalArgumentException("Invalid argument type. Expected payloadData " +
+                            "(Map<String, Object>), headers (Map<String, String>), and eventHandlers " +
+                            "(Map<String, Object>) respectively.");
+                }
```

Comment:
This message doesn't seem to tally with the arguments of the function.
Looks like the order is, headers, authentication config and event handlers

Reply:
Resolved with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/157/commits/01b5e2f5f849d7a4e89d792e28845597544fca39